spec/system/show_pdc_describe_record_spec.rb

```diff
@@ -15,12 +15,17 @@
       .to_return(status: 200, body: File.open(Rails.root.join("spec", "fixtures", "files", "pdc_describe_data", "works.rss")).read, headers: {})
     indexer = DescribeIndexer.new(rss_url: "http://pdc_test_data/works.rss")
     indexer.index
-    page.driver.browser.manage.window.resize_to(4000, 4000)
+    # page.driver.browser.manage.window.resize_to(4000, 4000)
```

Comment:
I am sorry, but was this meant to be un-commented or removed?

Reply:
Great catch, this line has been removed. Thanks!